editor/code_editor.cpp

```diff
@@ -1798,9 +1802,12 @@ CodeTextEditor::CodeTextEditor() {
 	status_bar->set_h_size_flags(SIZE_EXPAND_FILL);
 	status_bar->set_custom_minimum_size(Size2(0, 24 * EDSCALE)); // Adjust for the height of the warning icon.
 
+	idle_time = EDITOR_GET("text_editor/completion/idle_parse_delay");
+	idle_time_with_errors = EDITOR_GET("text_editor/completion/idle_parse_delay_in_error_state");
 	idle = memnew(Timer);
 	add_child(idle);
 	idle->set_one_shot(true);
+	idle->set_wait_time(idle_time);
```

Comment:
But then this should be somewhere else I suppose.

Reply:
But where?

Reply:
You can call it in `update_editor_settings()` if the time was 0.

Reply:
In hindsight it's not that big of a deal to reset the `idle` Timer when editor settings are modified, but I will still do the change.